tools/test-proxy/Azure.Sdk.Tools.TestProxy/RecordingHandler.cs

```diff
@@ -200,10 +201,10 @@ private byte[] DecompressBody(byte[] incomingBody, HttpContentHeaders headers)
                 if (values.Contains("gzip"))
                 {
                     using (var compressedStream = new MemoryStream(incomingBody))
-                    using (var zipStream = new GZipStream(compressedStream, CompressionMode.Decompress))
+                    using (var unzippedStream = new GZipStream(compressedStream, CompressionMode.Decompress))
```

Comment:
```suggestion
                    using (var uncompressedStream = new GZipStream(compressedStream, CompressionMode.Decompress))
```

NIT: Use name consistent with `CompressBody`